src/test/api/karate/businessconfig/resources/packageBundles.sh

```diff
@@ -13,4 +13,12 @@ cd ..
 cd bundle_api_test_apogee 
 tar -czvf bundle_api_test_apogee.tar.gz config.json i18n.json css/ template/ 
 mv bundle_api_test_apogee.tar.gz ../
+cd ..
+cd bundle_defaultProcess_V1
+tar -czvf bundle_defaultProcess_V1.tar.gz config.json i18n.json css/ template/
+mv bundle_defaultProcess_V1.tar.gz ../
+cd ..
```

Comment:
Must we really copy all files , could we just take the file form the resource directory ?

Reply:
Done.